packages/grid/_modules_/grid/components/Watermark.tsx

```diff
@@ -4,11 +4,11 @@ import { useLicenseVerifier, LicenseStatus } from '@mui/x-license-pro';
 function getLicenseErrorMessage(licenseStatus: LicenseStatus) {
   switch (licenseStatus) {
     case LicenseStatus.Expired:
-      return 'MUI X License Expired';
+      return 'MUI X License key expired';
```

Comment:
```suggestion
      return 'MUI X: License key expired';
```

Should this be "MUI X", or "Data Grid Pro"? If in the future someone has more than one component installed, it might be helpful to know which one is generating the error.

Reply:
This is for the content of the watermark, the developers can see this visually. I actually wonder how we will approach this problem with the date range picker cc @flaviendelangle